src/com/jambit/Database.java

```diff
@@ -0,0 +1,48 @@
+package com.jambit;
+
+import java.sql.*;
+
+public class Database {
+  private static final String HOST = "azubipi1.client.jambit.com";
+  private static final String USERNAME = "root";
+  private static final String PASSWORD = "123";
+  private static final String DATABASE = "new_schema";
```

Comment:
new_schema is not a proper name...

Reply:
The database is not fully set up at the moment. For testing purposes I took a already existing database.